app/src/androidTest/java/org/kiwix/kiwixmobile/webserver/ZimHostFragmentTest.kt

```diff
@@ -160,6 +160,10 @@ class ZimHostFragmentTest {
         // Check that only one ZIM file is hosted on the server
         assertItemHostedOnServer(1)
 
+        // Check Share by QR code
+        tapShareByQr()
+        assertQRDialogShown()
```

Comment:
The test case is failing due to the QR code showing on the screen, please add the `pressBack()` after `assertQRDialogShown()` so that the QR code will remove from the screen and the test case will pass.